test/pensions/servedDuringWartime.unit.spec.js

```diff
@@ -0,0 +1,32 @@
+import { expect } from 'chai';
+
+import { servedDuringWartime } from '../../src/js/pensions/helpers';
+
+describe('Wartime service validation', () => {
```

Comment:
Thanks for your recommendation to create these tests @jbalboni, please let me know if additional changes would make sense (naming, additional/different test cases, etc.).